Mixpanel/MPABTestDesignerConnection.m

```diff
@@ -199,14 +203,13 @@ - (void)webSocket:(MPWebSocket *)webSocket didReceiveMessage:(id)message
 {
     if (!_connected) {
         _connected = YES;
-        [self showConnectedView];
+        [self showConnectedView:NO];
```

Comment:
Could we rename this to `setConnectedViewHidden:`? I think it fits better with the `UIKit` API design.